doc/userguide/support-status.rst

```diff
@@ -0,0 +1,490 @@
+
+   +-------------+-------------+---------------------------+---------------------------------------------+
+   |Architecture |Support      |QA                         |Notes                                        |
+   +=============+=============+===========================+=============================================+
+   |PPC64el      |             |Part of Fedora automated QA|Access can be arranged through IBM dev cloud |
+   +-------------+-------------+---------------------------+---------------------------------------------+
+   |PPC64        |Victor Julien|                           |No access except an old Mac G5 Victor has    |
+   +-------------+-------------+---------------------------+---------------------------------------------+
+   |PPC32        |Victor Julien|                           |No access except an old Mac G4 Victor has    |
+   +-------------+-------------+---------------------------+---------------------------------------------+
+
+High Level Features
+-------------------
+
+Capture support
+~~~~~~~~~~~~~~~
+
+Tier 1
+^^^^^^
+
+.. table::
+   :width: 100%
+
+   +----------------+-------------------------+----+-----------------------------+
+   | Capture Type   | Maintainer              | QA | Notes                       |
+   +================+=========================+====+=============================+
+   |AF_PACKET       |Core team / Eric Leblond |    |Used by Security Onion, SELKS|
+   +----------------+-------------------------+----+-----------------------------+
+   |NETMAP (FreeBSD)|Core team / Victor Julien|    |Used by OPNsense             |
+   |                |+ Aleksey Katargin       |    |                             |
+   +----------------+-------------------------+----+-----------------------------+
+   |NFQUEUE         |Core team / Andreas Herz |    |                             |
+   +----------------+-------------------------+----+-----------------------------+
+   |libpcap         |Core team                |    |                             |
+   +----------------+-------------------------+----+-----------------------------+
+
+Tier 2
+^^^^^^
+
+.. table::
+   :width: 100%
+
+   +--------------------+-------------------------+----+---------------+
+   |Capture Type        |Maintainer               |QA  |Notes          |
+   +====================+=========================+====+===============+
+   |PF_RING             |Core team                |    |               |
+   +--------------------+-------------------------+----+---------------+
+   |AF_PACKET (eBPF/XDP)|Eric Leblond             |    |Bleeding edge  |
+   |                    |                         |    |               |
+   +--------------------+-------------------------+----+---------------+
+   |NETMAP (Linux)      |Core team / Victor Julien|    |               |
+   |                    |+ Aleksey Katargin       |    |               |
+   +--------------------+-------------------------+----+---------------+
+
+Community
```

Comment:
wonder if it makes sense to split this between vendor and community, where a vendor would be about supporting their own hw, like napatech or endace

Reply:
I fully agree, it makes it also more clear that there are different parts of the community

Reply:
I've added a *Vendor* support tier, documented it and moved Napatech into a *Vendor* section.